roles/addons/prometheus_server/tasks/main.yml

```diff
@@ -30,6 +30,23 @@
     - monitoring.exporters.snmp_exporter.with_generator is defined      
     - monitoring.exporters.snmp_exporter.with_generator
 
+- name: "firewalld █ Add services to firewall's {{ prometheus_server_firewall_zone | default('public') }} zone"
+  firewalld:
+    zone: "{{ prometheus_server_firewall_zone | default('public') }}"
+    service: "{{ item }}"
+    immediate: "yes"
+    permanent: "yes"
+    state: enabled
+  when:
+    - ansible_facts.os_family == "RedHat"
+    - ep_firewall | default(false) | bool
+  loop:
+    - 9090/tcp
+    - 9093/tcp
+    - 9100/tcp
```

Comment:
`9100/tcp` might be the port for node_exporter. I expect node_exporter to be installed with the role prometheus_client (which means there is a defect in the role, not in this PR). Am I wrong?

Reply:
9100 removed from the list, already opened by prometheus client role